src/reporting/quicksight-custom-resource.ts

```diff
@@ -140,10 +146,12 @@ export function createQuicksightCustomResource(
           {
             name: 'startDate',
             timeGranularity: TimeGranularity.DAY,
+            defaultValue: tenYearsAgo,
```

Comment:
What's the purpose of default value for dataset parameter? Why using a large time range as default values?

Reply:
make sure customer can find data when they use these dataset in custom analysis at default. Robin met this issue and proposed this change.